### ISSUE TITLE
Domain name and macOS Bundle Identifier

### ISSUE THREAD

**Issue by kbdluv** [Wilfred/remacs#586]
In 7153765199fbb57fd87091869050b6dfce19637f the `org.gnu.Emacs` bundle identifier gets renamed to `org.gnu.Remacs` in what is likely just a string replacement. This is obviously not an urgent issue, but leaving it as a placeholder for when there is a proper Remacs domain name to replace that, as it is unlikely that GNU will either adopt or merge the project soon.

**Comment by Wilfred** [comment]
Agreed, that seems reasonable. I suppose we could use `com.github.Remacs`. Is there a precedent for other open source projects -- what do they use?

**Comment by kbdluv** [comment]
I've found [one occurrence](https://github.com/jto/validation) of `io.github.projectName`. I like `org.remacs.remacs` or `org.remacs.emacs` better though, or even `uk.me.wilfred.remacs` until you decide Remacs is mature/has community enough for its own github org/domain name (which can fairly be never, as Spacemacs exemplifies). Just can't think of a reason to tie it go Github.
